examples/asr/speech_to_text_eval.py

```diff
@@ -144,6 +167,8 @@ def main(cfg: EvaluationConfig):
     else:
         logging.info(f'Got {metric_name} of {metric_value}')
 
+    logging.info(f'Dataset WER/CER ' + str(round(100 * wer, 2)) + "%/" + str(round(100 * cer, 2)) + "%")
```

Comment:
do we need to have both wer and cer here?

Reply:
I think it is better to see both metrics in one shot

Reply:
then how about merge line 170 with above if else condition and reduce redundant logging

Reply:
done